### PR TITLE
[setup] pin setuptools to a higher version

### LINKED ISSUE #3: Library has broken Python 2 support
On your project page, your PyPI badge indicates Python 2 support. However, trying to install it leads to a failure.

```
$ pip install librouteros
Collecting librouteros
  Downloading librouteros-1.0.1.tar.gz
    Complete output from command python setup.py egg_info:
    error in librouteros setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-MbOonr/librouteros/
```

Environment information:

```
$ python -V
Python 2.7.9
$ pip --version
pip 9.0.1 from /home/user/.virtualenvs/ros/local/lib/python2.7/site-packages (python 2.7)
```

### PR DESCRIPTION
**Pull request by froztbyte**
This closes #3 (and more information is provided in librouteros-docs#1)

### CODE REVIEW

**Comment by froztbyte** [comment]
@luqasz it seems travis is having issues with verifying the setuptools version requirements here (with setuptools failing). Do you have any specific advice for an approach to follow?

**Comment by froztbyte** [comment]
I'll attempt to test it in a venv on another platform.

**Comment by luqasz** [comment]
Try to print setuptools version on travis and pin its minimum. All tests passes normaly on travis so that version will be ok.

**Comment by luqasz** [comment]
`setuptools (12.0.5)` is installed on travis, so with this version and above should work.

File: setup.py [hunk 1]
```diff
@@ -23,9 +23,16 @@
         'bumpversion',
         )
 
+setup_pkgs = [
+    'packaging >= 16.8',
+    'setuptools >= 34.1.1',
```
Comment (luqasz) [inline setup.py]: only setuptools needs to be installed.

**Comment by luqasz** [comment]
Done in 0bb82c7cf44c8e8091151119f3b262a5556de956

**Pull request closed without merging**